CONTRIBUTING.md

```diff
@@ -13,6 +13,10 @@ https://qiskit.org/documentation/contributing_to_qiskit.html
 In addition to the general guidelines there are specific details for
 contributing to qiskit-ibm-runtime, these are documented below.
 
+### Open an issue
```

Comment:
I wasn't sure where to add the note in contributing.md since there isn't an explicit section about opening issues, so I added a section here. Please feel free to suggest other options...